autoload/ale/fixers/isort.vim

```diff
@@ -2,24 +2,35 @@
 " Description: Fixing Python imports with isort.
 
 call ale#Set('python_isort_executable', 'isort')
-call ale#Set('python_isort_options', '')
 call ale#Set('python_isort_use_global', get(g:, 'ale_use_global_executables', 0))
+call ale#Set('python_isort_options', '')
+call ale#Set('python_isort_auto_pipenv', 0)
+
+function! ale#fixers#isort#GetExecutable(buffer) abort
+    if (ale#Var(a:buffer, 'python_auto_pipenv') || ale#Var(a:buffer, 'python_isort_auto_pipenv'))
+    \ && ale#python#PipenvPresent(a:buffer)
+        return 'pipenv'
+    endif
+
+    return ale#python#FindExecutable(a:buffer, 'python_isort', ['isort'])
+endfunction
 
 function! ale#fixers#isort#Fix(buffer) abort
     let l:options = ale#Var(a:buffer, 'python_isort_options')
 
-    let l:executable = ale#python#FindExecutable(
-    \   a:buffer,
-    \   'python_isort',
-    \   ['isort'],
-    \)
+    let l:executable = ale#fixers#isort#GetExecutable(a:buffer)
+
+    let l:exec_args = l:executable =~? 'pipenv$'
+    \   ? ' run isort'
+    \   : ''
 
-    if !executable(l:executable)
+    if !executable(l:executable) && l:executable != 'pipenv'
```

Comment:
The current CI failure is complaining about `!=` here: `autoload/ale/fixers/isort.vim:27:50: Use robust operators `!=#` or `!=?` instead of `!=` (see Google VimScript Style Guide (Matching))`

Reply:
Looks like it also wanted `isnot` in place of `!=`, more robust when one of the operands isn't a string